CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Moved to Komposable organization ([#13](https://github.com/komposable/dekorator/pull/13))
 - Replace Travis CI by Github Actions and remove ruby 2.3 support ([#23](https://github.com/komposable/dekorator/pull/23))
+- Update railtie to not trigger initialization autoloaded constant deprecation warning ([#30](https://github.com/komposable/dekorator/pull/30))
```

Comment:
```suggestion
- Update railtie to prevent triggering initialization autoloaded constant deprecation warning ([#30](https://github.com/komposable/dekorator/pull/30))
```